docs/Config-Specification.rst

```diff
@@ -897,17 +897,17 @@ node_templates:
 |             pxe:                   |               interface: eth15                | |   *interface* - Reference to interface label defined in the `interfaces:`_     |          |
 |                 - switch:          |               rename: true                    |                elements.                                                         |          |
 |                   interface:       |               ports:                          | |   *rename* - Value (true/false) to control whether client node interfaces will |          |
-|                   rename:          |                   - 10                        |                be renamed to match the inerface label value.                     |          |
+|                   rename:          |                   - 10                        |                be renamed to match the inerface iface or DEVICE value.           |          |
 |                   ports:           |                   - 11                        | |   *ports*  - List of port number/identifiers mapping to client node OS         |          |
 |             data:                  |                   - 12                        |                interfaces.                                                       |          |
 |                 - siwtch:          |         data:                                 |                                                                                  |          |
-|                   interface:       |             - switch: data_1                  | Note: If *rename* is set to True, the physical interface name will be changed    |          |
-|                   rename:          |               interface: eth10                |       irregardless of whether it's original name matches the *iface* name        |          |
```

Comment:
These interface: values should be changed to one of the top level interface labels.  (ie manual1)  as is, they actually call out iface values which is likely to cause confusion.

Reply:
done in eb4f677